go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/giantswarm/cluster-test-suites
 
 go 1.21
 
+replace github.com/giantswarm/cluster-standup-teardown => github.com/giantswarm/cluster-standup-teardown v1.0.2-0.20240429091344-f18b62b0c51a
```

Comment:
This will need removing. I just wanted to test out the change before I created a new release.